src/test/java/com/endava/soa_5/cucumber/hook/Hooks.java

```diff
@@ -28,5 +31,6 @@ public void afterScenario(Scenario scenario) {
         log.info("---------------------END OF SCENARIO--------------------");
         log.info(format("ENDING TEST: [%s] %s ", scenario.getStatus().toString().toUpperCase(), scenario.getName()));
 //        log.info("*********************" + scenarioStatus + "*********************");
+        driver.quit();
```

Comment:
You're aware that it will close the driver after each scenario?

Reply:
We are still looking for a method to solve the problem with the driver. I would like to merge in finaltask_master because we have other dependencies. And we solve the problem with the driver in the next PRs